zstd.go

```diff
@@ -161,3 +161,18 @@ func Decompress(dst, src []byte) ([]byte, error) {
 	defer r.Close()
 	return ioutil.ReadAll(r)
 }
+
+// DecompressInto decompresses src into dst. Unlike Decompress, DecompressInto
```

Comment:
could you have `Decompress` calls `DecompressInto` at l.146 so we don't have to repeat the call to `C.ZSTD_decompress` ?

Reply:
Done.